src/rpcblockchain.cpp

```diff
@@ -1566,52 +1313,6 @@ UniValue resetcpids(const UniValue& params, bool fHelp)
     return res;
 }
 
-UniValue rsa(const UniValue& params, bool fHelp)
-{
-    if (fHelp || params.size() != 0)
-        throw runtime_error(
-                "rsa\n"
-                "\n"
-                "Displays RSA report for your CPID\n");
-
-    UniValue res(UniValue::VARR);
-
-    if (msPrimaryCPID.empty() || msPrimaryCPID == "INVESTOR")
-        throw runtime_error(
-                "PrimaryCPID is empty or INVESTOR; No RSA available for this condition\n");
-
-    LOCK(cs_main);
-
-    res = MagnitudeReport(msPrimaryCPID);
-
-    return res;
-}
-
-UniValue rsaweight(const UniValue& params, bool fHelp)
-{
-    if (fHelp || params.size() != 0)
-        throw runtime_error(
-                "rsaweight\n"
-                "\n"
-                "Display Rsaweight for your CPID\n");
-
-    UniValue res(UniValue::VOBJ);
-
-    double out_magnitude = 0;
-    double out_owed = 0;
-
-    LOCK(cs_main);
-
-    int64_t RSAWEIGHT = GetRSAWeightByCPID(GlobalCPUMiningCPID.cpid);
-    out_magnitude = GetUntrustedMagnitude(GlobalCPUMiningCPID.cpid, out_owed);
-
-    res.pushKV("RSA Weight", RSAWEIGHT);
-    res.pushKV("Magnitude", out_magnitude);
-    res.pushKV("RSA Owed", out_owed);
```

Comment:
Merge the owed field into `MagnitudeReport`.

Reply:
its untrustedmagnitude here. does the fulfillment in MagnitudeReport not more accurate? 
                        // Fulfillment %
                        double fulfilled = ((structMag.payments/14) / ((dExpected14/14)+.01)) * 100;
                        entry.pushKV("Fulfillment %", fulfilled);

Reply:
could figure out difference and name is estimated rsa owed?

Reply:
Since it's just for display we can use `"owed": structMag.owed`. I think.